cura_app.py

```diff
@@ -22,6 +22,7 @@
 # tries to create PyQt objects on a non-main thread.
 import Arcus  # @UnusedImport
 import Savitar  # @UnusedImport
+import pynest2d # @UnusedImport
```

Comment:
Why is this needed?

Reply:
For the same reason as the others are there, we get some weird sip pointer error if we don't.